metrics/src/main/java/com/lightstep/tracer/metrics/GcMetricGroup.java

```diff
@@ -0,0 +1,31 @@
+package com.lightstep.tracer.metrics;
+
+import java.io.IOException;
+import java.lang.management.GarbageCollectorMXBean;
+import java.lang.management.ManagementFactory;
+
+import oshi.hardware.HardwareAbstractionLayer;
+
+class GcMetricGroup extends MetricGroup {
+  GcMetricGroup(final HardwareAbstractionLayer hal) {
+    super(hal, new GaugeMetric<>("runtime.java.heap_size", Long.class), new CounterMetric<>("runtime.gc.count", Long.class), new CounterMetric<>("runtime.gc.time", Long.class));
```

Comment:
can you change this to `runtime.java.gc.count` and `runtime.java.gc.time` ? I'm sorry I missed it in the metrics questionnaire!